community/people/klyg/index.md

```diff
@@ -0,0 +1,18 @@
+---
+layout: person
+title: Katie Lygre
+image: lygre.jpeg
+role: PhD Student
+email: 
+  - lygre@wisc.edu
```

Comment:
Do you have an [ORCID](https://orcid.org/)? If not, I'd suggest setting one up now.

Reply:
I will look into that!

Reply:
More generally, in addition to ORCID, you can add various "services" e.g.
```
services:
  github: katielygre
  linkedin: katie-lygre-4a9073203
```
I won't hold up the merge for this, but it can be added later